packages/test-generator/integration-test-templates/cypress/e2e/create-form-spec.cy.ts

```diff
@@ -123,9 +123,26 @@ describe('CreateForms', () => {
         });
         clickAddToArray();
 
+        // ManyToMany Autocomplete
+        getArrayFieldButtonByLabel('Many to many tags').click();
+        cy.get(`.amplify-autocomplete`).within(() => {
+          cy.get('input').type(`Red{downArrow}{enter}`);
+        });
+        clickAddToArray();
+        getArrayFieldButtonByLabel('Many to many tags').click();
+        cy.get(`.amplify-autocomplete`).within(() => {
+          cy.get('input').type(`B{downArrow}{enter}`);
+        });
+        clickAddToArray();
+        getArrayFieldButtonByLabel('Many to many tags').click();
+        cy.get(`.amplify-autocomplete`).within(() => {
+          cy.get('input').type(`Gr{downArrow}{enter}`);
+        });
+        clickAddToArray();
+
         cy.contains('Submit').click();
 
-        cy.contains(/MyString/).then((recordElement) => {
+        cy.contains(/MyString/).then((recordElement: JQuery) => {
```

Comment:
I am not seeing the part where you check that the tags got saved?

Reply:
Will update